### ISSUE TITLE
Dispkay not mobile devices not working as expected

### ISSUE THREAD

**Issue by unaor** [primefaces/primeflex#39]
Im using "primeflex": "^2.0.0" and im trying to hide an element inside a p-grid on smaller screens

`<div class="p-col p-d-sm-none p-d-md-none"><img [src]="clientLogo" /></div>`

however what im noticing is that on screen smaller than 576PX, the element is displayed

i've check the file primeflex.css and noticed the following

```
@media screen and (min-width: 576px) {
.p-d-sm-none {
    display: none !important;
  }
}
```

as most mobile screens are smaller than this is there another selector to target them?

**Comment by mauricionr** [comment]
any update on this?

**Comment by developithecus** [comment]
I ran into the same thing. I find PrimeReact/primeflex fantastic, but given this particular issue, how would we even use it for proper responsive apps?

I don't get how this issue is still unresponded after almost 2 years. Seems critical for a layout library to me.

**Comment by melloware** [comment]
@developithecus are you using Primeflex 2 or 3 ?  Can you post a code sandbox reproducer or provide your example code here so I can see it and debug?

**Comment by developithecus** [comment]
Hello, @melloware
I'm using Primeflex 3.

I succeeded in fixing my issue, but didn't get around to updating my reply. I fiddled around with the classes some more after posting that, and found out the actual behaviour. It looks like I just had to have a change of mindset.

Seems like the breakpoints were behaving the opposite way than I was expecting. Here's what I was doing:
I was trying to make this div's width responsive and was expecting to get a full-width on mobile displays.
```html
<div class="w-8 sm:w-full" />
```
However, contrary to my expectations, the full width would only apply to screens larger than the given threshold, not smaller. 
Basically, I had to rewrite it as such:
```html
<div class="w-full sm:w-8" />
```
so that everything under `$sm` level would get the `w-full`, while everything above would get the `w-8`.

Am I wrong in saying this is the opposite of other UI libraries like bootstrap? IIRC, they work with `max-width` on media queries.

**Comment by melloware** [comment]
So @developithecus 

1. it sounds like first of this issue is not an issue and you got eveyrthing working?

2. It sounds like its just a difference of how Bootstrap does maxwidth and PF does minwidth for determining classes?  

As to why PF chose to do max-width I have no idea.   Its a good question for PrimeTek.  Maybe ask on the PrimeFlex Discord channel!

**Comment by mertsincan** [comment]
Hi, 

No, Tailwind and PrimeFlex use min-width media properties.  PrimeFlex a mobile-first breakpoint system, similar to what you might be used to in other frameworks like Bootstrap, Tailwind.

For first issue, you can override $breakpoints variable; https://github.com/primefaces/primeflex/issues/52#issuecomment-1069096707

Best Regrards,
